docs/rules/explicit-length-check.md

```diff
@@ -0,0 +1,21 @@
+# Enforce explicitly checking the length of an array
```

Comment:
This applies to any value with a `.length` property though. Like `string`, `TypedArray`, `Buffer`.